go.mod

```diff
@@ -4,11 +4,12 @@ go 1.13
 
 require (
 	github.com/crossplane/crossplane v0.13.0-rc.0.20200923162121-6e6cee50d87f
-	github.com/crossplane/crossplane-runtime v0.9.1-0.20201001234308-3f1afd106a8c
-	github.com/crossplane/crossplane-tools v0.0.0-20201001224552-fb258cc0eb30
+	github.com/crossplane/crossplane-runtime v0.9.1-0.20201008065523-51c117eff562
+	github.com/crossplane/crossplane-tools v0.0.0-20201007233256-88b291e145bb
 	github.com/google/go-cmp v0.4.0
 	github.com/pkg/errors v0.9.1
 	github.com/rook/rook v1.1.2
+	golang.org/x/tools v0.0.0-20200410194907-79a7a3126eef // indirect
```

Comment:
I do see this in the `crossplane-runtime` `go.mod`, but this specific bump doesn't seem like it would introduce it, particularly given the fact that https://github.com/crossplane/provider-aws/pull/373 did not add it to `provider-aws`. Not 100% sure though. Obviously this passes `check-diff` so having the indirect does work...

Reply:
I think this indirect is actually expected.